front/components/assistant_builder/actions/WebsearchAction.tsx

```diff
@@ -0,0 +1,18 @@
+import type { AssistantBuilderActionConfiguration } from "@app/components/assistant_builder/types";
+
+export function isActionWebsearchValid(
+  action: AssistantBuilderActionConfiguration
+) {
+  return (
+    action.type === "WEBSEARCH" && Object.keys(action.configuration).length > 0
+  );
+}
+
+export function ActionWebsearch() {
+  return (
+    <div>
+      This action will search the web and provide the 8 first results (title,
```

Comment:
"8 top results" but I would really say "the top results"

"provide" => "return"

"search" => "perform a search"